### PR TITLE
restructure tests and enable 100% test coverage check

### LINKED ISSUE #98: Coverage 100%
Increase coverage to reach 100%

### PR DESCRIPTION
**Pull request by Uzlopak**
Resolves #98 

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by Uzlopak** [comment]
This is basically a follow up to @anthonyringoet PR #139

File: .taprc [hunk 1]
```diff
@@ -1 +1 @@
-check-coverage: false
+check-coverage: true
```
Comment (climba03003) [inline .taprc:1]: I would just remove the file. Check coverage is `true` by default.

Comment (Uzlopak) [reply .taprc]: removed

**Review by climba03003 — APPROVED** [review]
LGTM.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**